### ISSUE TITLE
Add Krazo Module for Apache Tomcat

### ISSUE THREAD

**Issue by ivargrimstad** [eclipse-ee4j/krazo#351]

**Comment by erdlet** [comment]
We can find a up-to-date working example for Krazo on Tomcat here: https://github.com/mvc-toolbox/mvc-toolbox/blob/3d544396e564a197d81165851e4000ee38b8bdb5/showcase/pom.xml#L85-L178

**Comment by ivargrimstad** [comment]
> We can find a up-to-date working example for Krazo on Tomcat here: https://github.com/mvc-toolbox/mvc-toolbox/blob/3d544396e564a197d81165851e4000ee38b8bdb5/showcase/pom.xml#L85-L178

Yes, so we should package either a distributable JAR with all deps needed, or a BOM or something. To offer a set of deps that we can verify works and run the TCK against

**Comment by pizzi80** [comment]
Tomcat 10.0.23 is very old and in EOL since 1 year

https://tomcat.apache.org/tomcat-10.0-eol.html

IMO rest and mvc are core java web technologies and should be Tomcat / Jetty first 
and an example should be in the github home page of the project ... ;)
